README.md

```diff
@@ -40,19 +45,26 @@ from django import forms
 from prices.forms import Money
 
 from django_prices.forms import MoneyField
-from django_prices.Validators import (
+from django_prices.validators import (
     MaxMoneyValidator, MinMoneyValidator, MoneyPrecisionValidator)
 
+
 class DonateForm(forms.Form):
-    donator_name = forms.CharField(label='Your name')
+    donator_name = forms.CharField(label="Your name")
     donation = MoneyField(
-        label='net', currency='EUR', max_digits=9, decimal_places=2,
-        validators=[MoneyPrecisionValidator('EUR'),
-                    MinMoneyValidator(Money(5, 'EUR')),
-                    MaxMoneyValidator(Money(500, 'EUR'))])
+        label="net",
+        available_currencies=[("BTC", "bitcoins"), ("USD", "US dollar")],
```

Comment:
How will these two work with validators configured to use EUR?

Reply:
It is a very good question; it brings me to the solution I've described in the comment below.